composer.json

```diff
@@ -14,10 +14,10 @@
         "php": ">=7.1.3",
         "ext-json": "*",
         "ext-dom": "*",
-        "nikic/php-parser": "^4.0",
+        "nikic/php-parser": "4.12.0",
```

Comment:
Is there a reason why you pinned this to 4.12? With this, someone using the tombstone library could use no other version of `nikic/php-parser`. So I'd want to change this back to `^4.0`

Reply:
nikic/php-parser made a change in the 4.13 release that's breaking the psalm tests. I locked it to 4.12.0.

Reply:
Then we should look into these psalm tests. They're likely not even related to your changes, so it would be different concern fixing them, one that's out-of-scope for this PR.

So please revert the change. Locking all users of the tombstone library into _that specific_ version of `nikic/php-parser` is not a good idea.